README.md

```diff
@@ -276,6 +276,29 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+## Integrations
+
+Support for incoming requests with [valyala/fasthttp](https://github.com/valyala/fasthttp):
+
```

Comment:
We should probably also add imports here so that users can use direct code from here for sample run.

Reply:
here: https://github.com/nicolascb/aws-xray-sdk-go/commit/177c39ea7ded5b9dd1a5be8ec48d801f9fa12ea1